src/inputs/TextFieldInput/index.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { ReactElement } from 'react';
+import TextFieldMui, { TextFieldProps } from '@material-ui/core/TextField';
+import styled from 'styled-components';
+
+export type TextFieldInputProps = {
+  id?: string;
+  name: string;
+  label: string;
+  error?: string | undefined;
```

Comment:
you don't need `| undefined`, `?` is enough

Reply:
done!